### ISSUE TITLE
[REQ] ShareWood

### ISSUE THREAD

**Issue by Portablelle** [Jackett/Jackett#4898]
Hello,

I don't know how to add support myself for new trackers so i'm here to find someone that could do it for the community.

I would like Jackett to support a new semi-private french generalist tracker. Sharewood.

You can access it via sharewood(.)tv, no need for invitation.

Thank you !

**Comment by garfield69** [comment]
so its semi-private or is it private with open invitation at the moment?
Are the downloads aways freeleech all the time or do users have to maintain their ratio?
I'm asking so I can set the appropriate flags.

**Comment by pipopop** [comment]
Hi all !
Hi Garfield,
It's a ratio tracker.
I'm not sur right now, but I think it's a private tracker. It's open now because it's a new tracker, but it will be closed soon.
Great thanks,

**Comment by Portablelle** [comment]
Hello @garfield69 !

It is semi-private. They told us that they will never close sign up, so it will never ask for invitations.
It a ratio tracker, you should maintain a 0.75 ratio, no freeleech !

Thank you :)

**Comment by xfouloux** [comment]
Thanks @garfield69 did not see that other issue =) !
Will try to add this one before the end of the week guys

**Comment by garfield69** [comment]
nah, you are missing the point, it's already been written ;-)

**Comment by xfouloux** [comment]
lol ! my bad thanks

**Comment by ilike2burnthing** [comment]
https://www.sharewood.tv/ - ERR_CONNECTION_TIMED_OUT

**Comment by ilike2burnthing** [comment]
back up

**Comment by dan3805** [comment]
i got this on radarr / sonarr 

Query successful, but no results were returned from your indexer. This may be an issue with the indexer or your indexer category settings.

1 month ago all was working great

**Comment by garfield69** [comment]
Yeah well, the world continues to change, so be prepared to make adjustments.

Just tested this, and it works for me.
Sonarr example
![image](https://user-images.githubusercontent.com/6933240/102318806-cbd5ad00-3fde-11eb-8552-7d7162e5d675.png)
Radarr example
![image](https://user-images.githubusercontent.com/6933240/102318981-16efc000-3fdf-11eb-92a5-ca882c34e730.png)

turn on your advanced options switch for both Sonarr and Radarr, edit the relevant apps' indexer and review your category settings
I'm using TV for Sonarr and 2000 for Radarr

Still got a problem? report back and we can try a deep dive and see if we can figure it out.

**Comment by dan3805** [comment]
<img width="543" alt="Capture d’écran, le 2020-12-17 à 00 44 21" src="https://user-images.githubusercontent.com/35934387/102449045-69da6f00-4001-11eb-9f94-45cbbc24627a.png">




I tried everything, even installing radarr with the nightly branch. I tried a lot of category combinations. without success
2000 only does not work either

**Comment by garfield69** [comment]
ok.
see https://github.com/Jackett/Jackett/wiki/Troubleshooting#how-to-provide-a-jackett-enhanced-log-for-a-ticket 
and we can take a look and see what Sonnar is asking for, what Jackett found, and see if there is a clue to what is going on.
Thanks.

**Comment by dan3805** [comment]
> ok.
> see https://github.com/Jackett/Jackett/wiki/Troubleshooting#how-to-provide-a-jackett-enhanced-log-for-a-ticket
> and we can take a look and see what Sonnar is asking for, what Jackett found, and see if there is a clue to what is going on.
> Thanks.

check my edit , it was the freeleech option that caused problems

**Comment by garfield69** [comment]
Ah. Well done for figuring it out!

**Comment by ilike2burnthing** [comment]
https://www.sharewood.tv/ - Erreur 500 - Erreur interne du serveur!

**Comment by garfield69** [comment]
using the browser I get this error after the login, but clicking the button at the bottom returned a working page from which torrent browsing and mail reading were working.
a TEST of the indexer also worked for me.

**Comment by garfield69** [comment]
https://www.sharewood.tv/login > 502 Bad Gateway nginx

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://www.sharewood.tv/ > ERR_CONNECTION_TIMED_OUT

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
Sharewood is scheduled to close at midnight 29th March 2026

**Comment by garfield69** [comment]
Sharewood has closed

**Comment by garfield69** [comment]
v0.24.1502
